tests/system_tests/psyclone/test_psyclone_system_test.py

```diff
@@ -145,7 +145,7 @@ def steps(self, config):
             config.build_output / 'kernel',
             # this second folder is just to test the multiple folders code, which was bugged. There's no kernels there.
             Path(__file__).parent / 'skeleton/algorithm',
-        ])
+        ], api="dynamo0.3")
```

Comment:
again, maybe use a constant?

Reply:
I've added this as a conftest fixture - seems a bit of an overkill, but I didn't want to add it to FAB's `constants.py` (I don't think Fab should contain PSyclone-specific constants), and conftest was already there ;)